mm2src/mm2_main/src/lp_wallet.rs

```diff
@@ -413,40 +413,45 @@ pub struct GetMnemonicResponse {
     pub mnemonic: MnemonicForRpc,
 }
 
-#[derive(Debug, Display, Serialize, SerializeErrorType)]
+#[derive(Debug, Display, Serialize, SerializeErrorType, EnumFromStringify)]
 #[serde(tag = "error_type", content = "error_data")]
-pub enum GetMnemonicError {
+pub enum WalletsStorageRpcError {
```

Comment:
`WalletsStorageRpcError` is not an accurate name as well, `MnemonicsError` can be a more general name if we will use it for multiple RPCs but changing the error name can break GUI if there is something that relies on it.

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2317#discussion_r1910773287